docker-compose.local.yml

```diff
@@ -27,6 +29,7 @@ services:
     volumes:
       - 'core-storage:/data'
       - 'reports-storage:/reports'
+      - 'leviathan:/run/leviathan'
```

Comment:
These changes should be made to both compose files if we actually want to go forward with this, rather than only being used for qemu.

Reply:
@klutchell Good point, I've updated this PR for the other compose file, but I haven't test it. Do you have a rig you could try these changes out on?